src/styles/partials/components/_search.scss

```diff
@@ -0,0 +1,34 @@
+.dg_search-container {
+  position: relative;
+  width: 100%;
+
+  .dg_search-btn {
+    display: block;
+    position: absolute;
+    right: $base-spacing-unit * 2;
+    top: $padding-small;
+    font-size: $base-font-size;
+    color: $black;
+    cursor: pointer;
+    background: none;
+    border: none;
+
+    i:hover {
+      color: $blue-lightest;
+    }
+  }
+}
+.dg_search-input {
+  display: block;
```

Comment:
per our chat, everything can be removed except the padding.

Reply:
all removed